lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -308,7 +308,7 @@ def _on_all_certificates_invalidated(self, event: AllCertificatesInvalidatedEven
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 22
+LIBPATCH = 23
```

Comment:
This seems like a breaking change unfortunately, should we bump the major version?